src/main/java/liquibase/ext/spanner/CloudSpanner.java

```diff
@@ -49,12 +57,26 @@ public boolean dataTypeIsNotModifiable(final String typeName) {
 
   @Override
   public String getDateLiteral(final String isoDate) {
-    String literal = super.getDateLiteral(isoDate);
+    String literal;
+
+    // Construct the literal based on whether it is a DATE or TIMESTAMP
     if (isDateTime(isoDate)) {
-      literal = "TIMESTAMP " + literal.replace(' ', 'T');
+      Date date;
+      try {
+        date = new ISODateFormat().parse(isoDate);
+        Instant instant = date.toInstant();
+        OffsetDateTime utcDateTime = instant.atOffset(ZoneOffset.UTC);
+        String formattedDate = utcDateTime.format(ISO_LOCAL_DATE);
+        String formattedTime = utcDateTime.format(ISO_LOCAL_TIME);
+        literal = "TIMESTAMP '" + formattedDate + "T" + formattedTime + "Z'";
+      } catch (ParseException e) {
+        throw new RuntimeException("Error parsing ISO date: " + isoDate, e);
```

Comment:
I think it would be better to do the same as the super class does in a case like this, which is to `return "BAD_DATE_FORMAT:" + isoDate;`. I don't know exactly why Liquibase does that here, but as the super class does not throw an exception for an invalid value, we should probably also not do in this override.

Reply:
did like super class